.github/workflows/slack-deployment-message.yml

```diff
@@ -79,6 +79,10 @@ jobs:
           if [ "${commit_pr_number}" == "null" ]; then
             commit_pr_number=$(echo "$commit_message" | pcregrep --only-matching=1 '^[^\n]+\(#(\d+)\)')
```

Comment:
just confirming my understanding: so this line will not result in an error, but instead will simply not match anything in the commit message if conventional commits are not used? in which case we'll go to the next `if` block

Reply:
I am pretty sure that is the case. I did some tests with this line aside and it just set the `commit_pr_number` to empty string. 

I'll double check once more if everything is alright.

Reply:
![image](https://user-images.githubusercontent.com/11679005/180249266-34dab66f-43ca-431b-b581-da6d885f98ea.png)